src/k8s-engine/k3sHelper.ts

```diff
@@ -826,6 +919,72 @@ export default class K3sHelper extends events.EventEmitter {
       console.error('Error uninstalling Traefik', ex);
     }
   }
+
+  /**
+   * Rancher Desktop's exposed `kubectl` utility is actually a wrapper around `kuberlr`,
+   * which guarantees that the actual true `kubectl` utility is compatible
+   * with the current version of kubernetes on the server.
+   *
+   * Calling `kubectl --context rancher-desktop cluster-info` is a good way to verify
+   * that the correct version of `kubectl` is available, or to let the user know there
+   * was a problem downloading it.
+   *
+   * @param version
+   */
+  async getCompatibleKubectlVersion(version: semver.SemVer): Promise<void> {
+    const commandArgs = ['--context', 'rancher-desktop', 'cluster-info'];
+
+    try {
+      const { stdout, stderr } = await childProcess.spawnFile(resources.executable('kubectl'),
+        commandArgs,
+        { stdio: ['ignore', 'pipe', 'pipe'] });
+
+      if (stdout) {
+        console.info(stdout);
+      }
+      if (stderr) {
+        console.log(stderr);
+      }
+    } catch (ex: any) {
+      console.error(`Error priming kuberlr: ${ ex }`);
+      console.log(`Output from kuberlr:\nex.stdout: [${ ex.stdout ?? 'none' }],\nex.stderr: [${ ex.stderr ?? 'none' }]`);
+      const pattern = /Right kubectl missing, downloading.+Error while trying to get contents of .+\/kubernetes-release/s;
+
+      if (pattern.test(ex.stderr)) {
+        const major = version.major;
+        const minor = version.minor;
+        const lowMinor = minor === 0 ? 0 : minor - 1;
+        const highMinor = minor + 1;
+        const homeDirName = os.platform().startsWith('win') ? '%HOME' : '~';
```

Comment:
Super minor nit: it's `%HOME%` (it's wrapped, not prefix).

```suggestion
        const homeDirName = findHomeDir();